File: Dsa-day19.py
from typing import List

class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        n = len(fruits)
        used_baskets = [False] * n
        unplaced_fruits_count = 0
        
        for i in range(n):
            fruit_size = fruits[i]
            placed = False
            for j in range(n):
                if not used_baskets[j] and baskets[j] >= fruit_size:
                    used_baskets[j] = True
                    placed = True
                    break
            
            if not placed:
                unplaced_fruits_count += 1
                
        return unplaced_fruits_count

File: Dsa-day19.cpp
#include <vector>

class Solution {
public:
    int numOfUnplacedFruits(std::vector<int>& fruits, std::vector<int>& baskets) {
        int n = fruits.size();
        std::vector<bool> usedBaskets(n, false);
        int unplacedFruitsCount = 0;
        
        for (int i = 0; i < n; i++) {
            int fruitSize = fruits[i];
            bool placed = false;
            for (int j = 0; j < n; j++) {
                if (!usedBaskets[j] && baskets[j] >= fruitSize) {
                    usedBaskets[j] = true;
                    placed = true;
                    break;
                }
            }
            
            if (!placed) {
                unplacedFruitsCount++;
            }
        }
        
        return unplacedFruitsCount;
    }
};